integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -611,20 +612,30 @@ def test_radians(data_gen):
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : unary_op_df(spark, data_gen).selectExpr('radians(a)'))
 
+def get_java_version():
+    jdk_version = check_output(['java', '-version'], stderr=STDOUT)
+    jdk_version = jdk_version.split(b'\n')[0].split(b' ')[2].decode('utf-8')
```

Comment:
This works well for the following version prints.

```
java version "1.8.0_25"
Java(TM) SE Runtime Environment (build 1.8.0_25-b17)
Java HotSpot(TM) 64-Bit Server VM (build 25.25-b02, mixed mode)

openjdk version "1.8.0_302"
OpenJDK Runtime Environment (build 1.8.0_302-bre_2021_07_21_23_11-b00)
OpenJDK 64-Bit Server VM (build 25.302-b00, mixed mode)
```

Nit:
Seems the following code is more readable:
```
        for line in output.splitlines():
            # E.g. openjdk version "1.8.0_212"
            line = line.strip()
            if 'version' in line:
                ver = line.split('version', 1)[1].strip()
                if ver.startswith('"') and ver.endswith('"'):
                    ver = ver[1:-1]
```

Reply:
Thanks, revised.